src/crafting.h

```diff
@@ -104,6 +104,84 @@ struct recipe {
 
 };
 
+/**
+*   enum used by comp_selection to indicate where a component should be consumed from.
+*/
+enum usage {
+    use_from_map = 1,
+    use_from_player = 2,
+    use_from_both = 1 | 2,
+    use_from_none = 4,
+    cancel = 5 // FIXME: hacky.
```

Comment:
Yes, I was wondering about that. The enum was locally defined in existing code, It already had the ```1 | 2```. I wanted to change the code as least as possible, so I kept the  ```1 | 2```. I thought it was just the same as 3. ```10 | 01 -> 11```.

Does defining the enum this way tell the compiler it can treat it's values differently than when defining just ```3```?